group.go

```diff
@@ -139,6 +139,35 @@ func (c *Group) Broadcast(route string, v interface{}) error {
 	return err
 }
 
+// Broadcast push  the message(s) to  all members
+func (c *Group) BroadcastExcludeId(route string, v interface{}, sID int64) error {
```

Comment:
You can implement this logic by Multicast, the filter is a predicate to check the session id.

Reply:
Thanks, I will use that instead. Removed from the PR.